openshift-install-powervs

```diff
@@ -1148,7 +1148,7 @@ function setup_terraform {
     if [[ "${ARCH}" == "ppc64le" ]]; then
       retry "curl --connect-timeout 30 -fsSL https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/terraform-${TF_LATEST:1} -o ${TF}"
     else
-      retry "curl --connect-timeout 30 -fsSL https://releases.hashicorp.com/terraform/${TF_LATEST:1}/terraform_${TF_LATEST:1}_${OS}_amd64.zip -o ./terraform.zip"
+      retry "curl --connect-timeout 30 -fsSL https://releases.hashicorp.com/terraform/${TF_LATEST:1}/terraform_${TF_LATEST:1}_${OS,}_amd64.zip -o ./terraform.zip"
```

Comment:
Is the ${OS,}_amd64.zip correct?

Reply:
@aishwaryabk can you explain why we need a comma (,)?